addons/mail/static/tests/message/message.test.js

```diff
@@ -1816,6 +1816,34 @@ test("Click on view reactions shows the reactions on the message", async () => {
     await contains(".o-mail-MessageReactionMenu", { text: "😅1" });
 });
 
+test("Reactions are in order by id", async () => {
+    const pyEnv = await startServer();
+    const channelId = pyEnv["discuss.channel"].create({
+        channel_type: "channel",
+        name: "channel1",
+    });
+    pyEnv["mail.message"].create({
+        body: "Hello world",
+        res_id: channelId,
+        message_type: "comment",
+        model: "discuss.channel",
+        reaction_ids: [
+            pyEnv["mail.message.reaction"].create({
+                content: "➕",
+                partner_id: serverState.partnerId,
+            }),
+            pyEnv["mail.message.reaction"].create({
+                content: "🔢",
+                partner_id: serverState.partnerId,
+            }),
+        ],
+    });
+    await start();
+    await openDiscuss(channelId);
+    await contains(":nth-child(1 of .o-mail-MessageReaction)", { text: "➕" });
+    await contains(":nth-child(2 of .o-mail-MessageReaction)", { text: "🔢" });
```

Comment:
- Prefer `:eq()` over `:nth-child()`, shorter is better
- VSC warns of ➕ being a source of error, so I changed to 🔰